tests/charm++/load_balancing/meta_lb_test/period_selection.C

```diff
@@ -41,10 +41,15 @@ public:
     if (iteration < MAX_ITER) {
       arrayProxy.balance(iteration);
     } else {
-      if (CkNumPes() > 1 && migrations != 10) {
+      int expected_migrations = 10;
+      if (CkNumPes() == 1) {
```

Comment:
Should this CkNumPes() and the one below be CkNumNodes()? If so, would that eliminate the need for the `#if CMK_UTH_VERSION` part?

Reply:
Nevermind, I bet within-node balancing is something we do.